telemetry/definitions/commonDefinitions.json

```diff
@@ -3,13 +3,13 @@
         {
             "name": "appRunnerServiceSource",
             "type": "string",
-            "allowedValues": ["ecr", "ecrPublic", "repository"],
+            "allowedValues": [ "ecr", "ecrPublic", "repository" ],
```

Comment:
Was this formatting done by  `prettier` as defined in https://github.com/aws/aws-toolkit-common/blob/f82e51208471921132e2b7a79c73116436ceb584/telemetry/vscode/.prettierrc ?

If not, this should probably be reverted because other developers' tools might flip-flop the formatting. If this was your editor or IDE (vscode), you might want to disable the auto-formatting.

Reply:
is this the only place formatting has happened? 
I have disabled my auto formatting.